identity-apps-core/pom.xml

```diff
@@ -734,7 +739,7 @@
         <carbon.extension.identity.authenticator.version>3.0.0</carbon.extension.identity.authenticator.version>
         <org.wso2.carbon.identity.association.account>5.1.5</org.wso2.carbon.identity.association.account>
         <identity.extension.utils>1.0.8</identity.extension.utils>
-        <carbon.identity.framework.version>7.7.176</carbon.identity.framework.version>
+        <carbon.identity.framework.version>7.8.146-SNAPSHOT</carbon.identity.framework.version>
```

Comment:
Bump the framework version after merging the following PR: https://github.com/wso2/carbon-identity-framework/pull/6732